### PR TITLE
added galaxy file with all the description

### PR DESCRIPTION
**Pull request by Vaishnavi-infoblox**

### CODE REVIEW

File: ansible_collections/infoblox/nios_modules/galaxy.yml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+tags:
+- infoblox
+- nios
+
+# Collections that this collection requires to be installed for it to be usable. The key of the dict is the
+# collection label 'namespace.name'. The value is a version range
+# L(specifiers,https://python-semanticversion.readthedocs.io/en/latest/#requirement-specification). Multiple version
+# range specifiers can be set and are separated by ','
+dependencies: 
+- infoblox-client
```
Comment (freedge) [inline ansible_collections/infoblox/nios_modules/galaxy.yml:50]: this syntax is incorrect apparently, yields

```
AttributeError: 'list' object has no attribute 'items'
```

Comment (Vaishnavi-infoblox) [reply ansible_collections/infoblox/nios_modules/galaxy.yml:50]: Could you please let me know what is the right syntax?

Comment (freedge) [reply ansible_collections/infoblox/nios_modules/galaxy.yml:50]: I just removed the whol
```
dependencies: 
- infoblox-client
```
block. From the comment above, it seems that this dependencies block pertain only to the "collections" you depend on, not the rest of the pip dependencies... personally I still maintain a requirements.yml for collections + a requirements.txt for pip dependencies.

Comment (Vaishnavi-infoblox) [reply ansible_collections/infoblox/nios_modules/galaxy.yml:50]: Sure I will also remove the same. Thank you Freedge

Comment (freedge) [reply ansible_collections/infoblox/nios_modules/galaxy.yml:50]: Thanks for your work! one last thing (just mentionning this here) the lookup does not seem performed through the collection. I use this:
```
  collections:
    - infoblox.nios_modules
...
    set_fact:
      lease: "{{ lookup('nios', 'lease', filter=nios_filter, provider=nios_provider, return_fields=nios_fields ) }}"
```

and this does not seem to be calling any code from the collection (unlike for example, calling "nios_fixed_address" that does indeed, uses the code from the collection. Do you know if there is something to do to use the collection for the lookup?

**Pull request merged**